src/Adapter/News/NewsDataProvider.php

```diff
@@ -140,26 +140,23 @@ public function getData($isoCode)
             if ($articles_limit == 0
                 || !$this->validate->isCleanHtml((string) $item->title)
                 || !$this->validate->isCleanHtml((string) $item->description)
-                || !isset($item->link, $item->title)) {
-                break;
+                || empty($item->link)
+                || empty($item->title)) {
+                continue;
```

Comment:
Special case for `$articles_limit == 0` you can `break`, it will be more efficient (although the result will be the same)

Reply:
Done